### ISSUE TITLE
insecure gradlew

### ISSUE THREAD

**Issue by IzzySoft** [martinchodev/Diaspora-Native-WebApp#60]
I'm just improving the metadata at F-Droid – and at that, your app's metadata got a minor adjustment – which caused our lint to complain:

```
repositories {
        maven {
            url  "http://dl.bintray.com/dasar/maven"
        }
gradle build uses plain HTTP URLs for repositories!  This is insecure!
https://max.computer/blog/how-to-take-over-the-computer-of-any-java-or-clojure-or-scala-developer/
```

As that URL redirects to `https` anyway: would you mind fixing that in your gradle directly? Makes it a bit safer :wink:

Thanks!

**Comment by dmorley** [comment]
This project was abandoned years ago

**Comment by IzzySoft** [comment]
@dmorley thanks, indeed looks a bit like that: no commit for about 1.5 years, and the "merged" project doesn't even exist anymore. Still, I'd like to hear that from a project member (and, if confirmed, would suggest "archiving" it so it's clear to whoever visits it) – as only then can we mark the project such at F-Droid (oh, if this repo would disappear it would be clear as well, of course). Any chance for an "official statement"?

**Comment by mdev88** [comment]
As @IzzySoft said, this project has been unnatended for a very long time. 
I don't think I should archive or delete this project. The original Diaspora app on which I based this app was like 5 years old when I started, so this might be useful in the future.
I think the readme file is clear enough about this app no longer being updated.

Cheers

**Comment by IzzySoft** [comment]
@mdev88 if you intend to work on it, keeping it open is useful. If not, better archive it (I agree to not delete); that still keeps the code available, and whoever wants to work on it has to fork it anyway. But yes, the decision is yours of course 😉

**Comment by mdev88** [comment]
Actually... I've been thinking about resurrecting this app in the last couple of weeks, so hold on ;)

**Comment by IzzySoft** [comment]
Just let me know then, so I can recheck. Thanks!

**Comment by mdev88** [comment]
Fixed in next release: version 1.9

**Comment by IzzySoft** [comment]
What? Ah, the Readme. OK, thanks for that! I was already wondering as no gradle file has been touched (see initial post), and no tag 1.9 could be found 😉

But why removing the "Latest APK" block? The directory is still there, and even holds an APK (with the same timestamp as the one on playstore). Did you rather mean to remove the F-Droid Link (which currently doesn't find the app, as it's no longer listed in the main repo)? The app has been moved to archive meanwhile, which has no official "web link" (you can use [this one](https://apt.izzysoft.de/fdroid/index/apk/ar.com.tristeslostrestigres.diasporanativewebapp?repo=archive) if you wish).

**Comment by mdev88** [comment]
I have not yet published the update, but it is fixed in version 1.9.
Once I publish the code here it will be updated on f-droid automatically.

**Comment by IzzySoft** [comment]
Ah, sure. I was just irritated because I didn't see those commits (as you didn't yet push them) :rofl:
